test/break_preserved_in_blockquote.md

```diff
@@ -1,5 +1,5 @@
 a
 
 > b  
-> c
+> c  
```

Comment:
Why does this result now have trailing whitespace? I wouldn't expect this test to change.

Reply:
Indeed, the current code makes all blockquotes to end with a linebreak.

This behavior comes from the handling of list-items wrapping:
https://github.com/Alir3z4/html2text/blob/master/test/wrap_list_items_example.md

Those trailing whitespaces don't seem very logical to me neither, or I am missing something?